### PR TITLE
Maz QA 8..10

### PR DESCRIPTION
**Pull request by rschlaikjer**

### CODE REVIEW

File: script/QA/QA_08.txt [hunk 1]
```diff
@@ -17,22 +17,22 @@ Why did the protagonist break into a locked room?
 -- Page 158, Offset 43051.
 -- 人類史における数々のミステリーに
 // 知能番組, so perhaps instead of "intellgence program" it could be more explicit "programme for thinkers"?, as that is kinda the implication here.
-The intelligence program that tackles countless mysteries
+The intellectual program that tackles countless mysteries%{n}
```
Comment (Avuxo) [inline script/QA/QA_08.txt:20]: these two lines don't flow right.
> The intellectual program that tackles _the_ countless mysteries of human history head_-_on

File: script/QA/QA_08.txt [hunk 2]
```diff
@@ -52,61 +52,61 @@ From here on, there will be many more death flags,
 [sha:0f535da664222ae1aef264145b0c0b8969c028ed]{
 -- Page 162, Offset 43058.
 -- 賢明な遠野くんにおかれましては、
-but for the wise and prudent Tohno-kun,
+but for the wise, prudent Tohno-kun,%{n}
```
Comment (Avuxo) [inline script/QA/QA_08.txt:55]: comma-ending lines end with 3x cjk dashes in en

File: script/QA/QA_08.txt [hunk 3]
```diff
@@ -52,61 +52,61 @@ From here on, there will be many more death flags,
...
 -- Page 164, Offset 43062.
 -- メガネともあろう男が油断しすぎだよニャー。
-It seems that glasses was too careless nya.
+Seems like Glasses didn't get the memo about being%{n}careful, nya.
 }
 [sha:c154c7fcdb92e39f7d1fb57d0bcc394f74dd7aeb]{
 -- Page 164, Offset 43063.
 -- それともあの包帯男、見かけよりやるってコト？
-Or perhaps that bandaged guy is more than he seems? 'is [capable of] more than he seems?' to catch the nuance of やる here, perhaps.
+ Or maybe that bandaged guy is more capable%{n}than he looks?
```
Comment (Avuxo) [inline script/QA/QA_08.txt:82]: first space used for glue or mistake? Same below

File: script/QA/QA_08.txt [hunk 4]
```diff
@@ -52,61 +52,61 @@ From here on, there will be many more death flags,
...
 }
 [sha:a2aeab14ecc83dc6d424a198c86fc976060a0f1a]{
 -- Page 165, Offset 43064.
 -- あの部屋の異様さに呑まれていた……のもありますが、
-As strange and captivating that room may be... the fault
+Strange and captivating though that room may be... %{n}the fault
```
Comment (Avuxo) [inline script/QA/QA_08.txt:87]: I prefer "As...as that may be" personally

File: script/QA/QA_08.txt [hunk 5]
```diff
@@ -141,23 +141,23 @@ Ciel is a martial artist,
...
 -- 遠野くんのフォローをしながら潜入……
 // the usage of フォロー (follow) in  Japanese has deviated quite a bit from its loanword root. This is better translated as 'support', 'back-up' or something like that.
-but following Tohno-kun and infiltrating...
+but supporting Tohno-kun while infiltrating...
 }
 [sha:578fca72909f2f131b733a1ccc7799164a88c9dd]{
 -- Page 172, Offset 43078.
 -- というのは、確かにわたし向きではありませんね。
-Frankly it would not suit me at all.
+Being quite honest, it would not suit me at all.%{n}
```
Comment (Avuxo) [inline script/QA/QA_08.txt:160]: revert and put comma after Frankly

File: script/QA/QA_09.txt [hunk 6]
```diff
@@ -177,38 +177,38 @@ It's not as long as the lecture on the vampire hierarchy.
...
 -- Page 197, Offset 43126.
 -- 清らかな男女交際トークです。
-between a lost junior and his reliable senior.
+ between a lost junior%{n}and his reliable senior.
 }
 [sha:ceb4ede87e45ce551a0cbf9f4b4002f991df605d]{
 -- Page 198, Offset 43127.
 -- 学校ならアタシも行ったし。
-If you're talking about school, I was there too.
+If you're talking about school,%{n}I was there too.
```
Comment (Avuxo) [inline script/QA/QA_09.txt:190]: This is a J->E antipattern. May as well translate it as something more appropriate like just:  "I went to school too"

File: script/QA/QA_10.txt [hunk 7]
```diff
@@ -1,90 +1,90 @@
...
 -- Page 212, Offset 43147.
 -- だいたい、痛くなくては覚えませんよね？
-Besides, deaths aren't memorable if they're not painful, right?
+Besides, deaths aren't memorable if they're%{n}not painful, right?
 }
 [sha:d62e5c5e12cda093550d4abf4329c9d3e7003e03]{
 -- Page 213, Offset 43148.
 -- ん～～、スパルタ通り越したドエスだった。
-Mmmm~~, you went right past Sparta and straight to Sadist.
+Mmmm~~, you went right past Sparta and straight%{n}to Sadist.
```
Comment (Avuxo) [inline script/QA/QA_10.txt:76]: How ruthless, Toshi!

File: script/QA/QA_10.txt [hunk 8]
```diff
@@ -1,90 +1,90 @@
...
 }
 [sha:ca3fc8b20fa87386dff096205c01f97983a1d876]{
 -- Page 214, Offset 43149.
 -- さて、今回の原因は『夜の』学校に
-Now then, the cause of death this time is due to
+Now then, the cause of death this time is due to%{n}
```
Comment (Avuxo) [inline script/QA/QA_10.txt:81]: due to:

File: script/QA/QA_10.txt [hunk 9]
```diff
@@ -157,69 +157,69 @@ If I had to guess, maybe it's because she finally
...
 -- 我慢できなくなっていたようですし？
 // Not Dead Apostles (使徒), but the Dead (死者)
-ever since they were at that garage full of dead apostles.
+after they were at that garage full of the Dead.
 }
 [sha:d568ced9455a4a49407807f94af248eb911e9c85]{
 -- Page 223, Offset 43167.
 -- マジかよそんな前から微熱（※かわいい）だったとか、
-Really? Did you feel your chances of winning
+Seriously? She was suffering from a fever (cuteness)%{n}even back then?
```
Comment (Avuxo) [inline script/QA/QA_10.txt:176]: bring back fun glyph ※

**Review by Avuxo — APPROVED** [review]
handful of changes

**Comment by rschlaikjer** [comment]
Comma -> CJK comments deferred, since pretty sure a lot of these are actually glued in-game. Hitting some of these in the next few days and will confirm then.

**Pull request merged**